src/builtin.cpp

```diff
@@ -3048,6 +3048,85 @@ static int builtin_bg(parser_t &parser, io_streams_t &streams, wchar_t **argv) {
     return res;
 }
 
+/// Helper for builtin_disown
+static int disown_job(parser_t &parser, io_streams_t &streams, job_t *j) {
+    if (j == 0) {
+        streams.err.append_format(_(L"%ls: Unknown job '%ls'\n"), L"bg");
+        builtin_print_help(parser, streams, L"disown", streams.err);
+        return STATUS_BUILTIN_ERROR;
+    }
+
+    // Stopped disowned jobs must be manually signalled; explain how to do so
+    if (job_is_stopped(j)) {
+        streams.err.append_format(
```

Comment:
This is likely to be my only substantive comment on this PR. My feeling is that we should break from bash/zsh and automatically send the SIGCONT to the job. Perhaps with a warning to the user that we did so. But I feel fairly strongly that the bash/zsh behavior is not friendly and we can do better. As I mentioned in another comment if I type `sleep 666` then press [ctrl-Z] followed by `disown` I expect that sleep command to be running in the background independent of the current fish shell. I should have to type `kill -SIGCONT -1234` to ensure the job continues to run.